cfpb/cfgov-refresh
# Reduce wagtail notification types to warning type only

## Changes

- Removes type field from wagtail notifications.
- Makes warning notification type the default in the notification template.

## Testing

1. Create a new sublanding page in wagtail and add a notification to the content area. ([Shortcut](http://localhost:8000/admin/pages/add/v1/sublandingpage/319/))
2. See that there is no longer a type drop-down and when you fill in the required fields and preview the page, the notification is of the warning type.
